transformers/integration_tests/vat_rely_test.go

```diff
@@ -54,11 +54,11 @@ var _ = Describe("Vat Rely transformer", func() {
 		err := db.Select(&dbResult, `SELECT usr FROM maker.vat_rely`)
 		Expect(err).NotTo(HaveOccurred())
 
-		usrAddress := "0x0e4725db88bb038bba4c4723e91ba183be11edf3"
+		usrAddress := "0xbaa65281c2fa2baacb2cb550ba051525a480d3f4"
 		usrAddressID, usrAddressErr := shared.GetOrCreateAddress(usrAddress, db)
 		Expect(usrAddressErr).NotTo(HaveOccurred())
 
-		Expect(len(dbResult)).To(Equal(1))
+		Expect(len(dbResult)).To(Equal(2))
 		Expect(dbResult[0].Usr).To(Equal(usrAddressID))
```

Comment:
Are we confident that the `usr` is the same for both events in this block?

Reply:
Its different in each event in this block but I've updated the test incase the ordering is off.